[0;33m--- Warning: GDB cannot open serial ports accessed as COMx[0m
[0;33m--- Using \\.\COM9 instead...[0m
[0;33m--- esp-idf-monitor 1.5.0 on \\.\COM9 115200[0m
[0;33m--- Quit: Ctrl+] | Menu: Ctrl+T | Help: Ctrl+T followed by Ctrl+H[0m
[0;33m--- 0x400495ea: ets_sha_process in ROM
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp84gd7s5_"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp84gd7s5_': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp50906ku6"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp50906ku6': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpnaa1yw4s"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpnaa1yw4s': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp03g1jaqt"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp03g1jaqt': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpqs54yu21"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpqs54yu21': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpocb848rv"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpocb848rv': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpco_9xqm6"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpco_9xqm6': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpstzjq5o3"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpstzjq5o3': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpbj87k701"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpbj87k701': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpd3i62wg4"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpd3i62wg4': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpz1rg_im5"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpz1rg_im5': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmplg0p9sqb"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmplg0p9sqb': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp6a89vxof"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp6a89vxof': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpouq27aip"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpouq27aip': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpq0ns1bjx"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpq0ns1bjx': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp94zh5by3"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp94zh5by3': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpb1xn1fxl"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpb1xn1fxl': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpu9dp6pdx"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpu9dp6pdx': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpy4gdh44i"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpy4gdh44i': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpaya9wpkg"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpaya9wpkg': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp2gbmkedl"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp2gbmkedl': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmppmtq_v2w"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmppmtq_v2w': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpusuoecjz"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpusuoecjz': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmps9uu84fn"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmps9uu84fn': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpqn4_kqn6"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpqn4_kqn6': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpqedf6lnv"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpqedf6lnv': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpw4zj2b_d"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpw4zj2b_d': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpc655yfad"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpc655yfad': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpori87zkl"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpori87zkl': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpbm3t4wyf"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpbm3t4wyf': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpx_jqbvfb"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpx_jqbvfb': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmphqbb_8dc"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmphqbb_8dc': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp8tkvrprn"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp8tkvrprn': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp3wczp33o"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp3wczp33o': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp8fg90dq5"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp8fg90dq5': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpufix_vty"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpufix_vty': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpjbmxtwv0"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpjbmxtwv0': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp0wtib6yp"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp0wtib6yp': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpv_j0g8h2"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpv_j0g8h2': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmphr4z190l"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmphr4z190l': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpr6dhja4y"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpr6dhja4y': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp5jkxofpd"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp5jkxofpd': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpx5co0qxc"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpx5co0qxc': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpvlf5vxic"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpvlf5vxic': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpcr05vm80"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpcr05vm80': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpbkwr_xr1"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpbkwr_xr1': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpmo3hycig"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpmo3hycig': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpmy0ybykt"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpmy0ybykt': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpnnlbjdnr"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpnnlbjdnr': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp8nyevq4w"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp8nyevq4w': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpl3osfdgc"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpl3osfdgc': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp98s81ttl"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp98s81ttl': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp0oyeosy2"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp0oyeosy2': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpgis0vs8t"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpgis0vs8t': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpiiek_gho"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpiiek_gho': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpv82cxuuu"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpv82cxuuu': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp91e6duj5"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp91e6duj5': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp6et504n8"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp6et504n8': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp93ybczef"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmp93ybczef': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpf39rz70i"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpf39rz70i': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpdgaabnal"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpdgaabnal': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpre8njyp5"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpre8njyp5': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Espressif\\frameworks\\esp-idf-v5.3.1\\esp-dev-kits\\esp32-c3-lcdkit\\examples\\knob_panel\\build\\c3_knob_panel.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c3 "C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpq8mh9m1f"', '-ex', 'bt']' returned non-zero exit status 1.
b"\r\nwarning: could not convert 'main' from the host encoding (CP1252) to UTF-32.\r\nThis normally should not happen, please file a bug report.\r\nerror starting child process 'C:\\Espressif\\python_env\\idf5.3_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c3 C:\\Users\\zzmmk\\AppData\\Local\\Temp\\tmpq8mh9m1f': CreateProcess: No such file or directory\r\nNo stack.\r\n"

[0m
[0;33m--- 0x403807d2: esp_restart_noos at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/port/soc/esp32c3/system_internal.c:110
[0m
[0;33m--- Stack dump detected[0m
[0;33m--- 0x4038081a: panic_abort at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/panic.c:463
0x4038756c: __ubsan_include at C:/Espressif/frameworks/esp-idf-v5.3.1/components/esp_system/ubsan.c:311
[0m
[0;33m--- 0x40380001: _vector_table at C:/Espressif/frameworks/esp-idf-v5.3.1/components/riscv/vectors_intc.S:54
[0m
[0;33m--- To exit from IDF monitor please use "Ctrl+]". Alternatively, you can use Ctrl+T Ctrl+X to exit.[0m
[0;33m--- To exit from IDF monitor please use "Ctrl+]". Alternatively, you can use Ctrl+T Ctrl+X to exit.[0m
[0m
Traceback (most recent call last):
  File "C:\Espressif\frameworks\esp-idf-v5.3.1\tools/idf_monitor.py", line 11, in <module>
    sys.exit(subprocess.run([sys.executable, '-m', 'esp_idf_monitor'] + sys.argv[1:]).returncode)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "subprocess.py", line 550, in run
  File "subprocess.py", line 1199, in communicate
  File "subprocess.py", line 1262, in wait
  File "subprocess.py", line 1543, in _wait
KeyboardInterrupt
